src/io/perun/asciidoctor.clj

```diff
@@ -0,0 +1,13 @@
+(ns io.perun.asciidoctor
+  (:require [io.perun.core :as perun]
+            [clojure.java.io :as io])
+  (:import [org.asciidoctor Asciidoctor Asciidoctor$Factory]))
+
+(defn asciidoctor-to-html [file-content]
+  (.convert (Asciidoctor$Factory/create "") file-content {}))
```

Comment:
This might not make much difference, but rather than creating the Asciidoctor interface on each call to `asciidoctor-to-html`, would it be more efficient to say, `(def asciidcotor (Asciidoctor$Factory/create ""))` as a top-level form, and then reuse the same object?